src/transformers/models/imagegpt/modeling_imagegpt.py

```diff
@@ -998,7 +998,7 @@ def forward(
         >>> samples = output[:, 1:].cpu().detach().numpy()
         >>> samples_img = [
         ...     np.reshape(np.rint(127.5 * (clusters[s] + 1.0)), [height, width, 3]).astype(np.uint8) for s in samples
-        ... ]  # convert color cluster tokens back to pixels
+        >>> ]  # convert color cluster tokens back to pixels
```

Comment:
Are these intended?

Reply:
No - I thought I removed those. Not sure why they came back 😂